client/src/components/timeline/Lane.vue

```diff
@@ -58,6 +59,10 @@ export default {
     };
   },
   methods: {
+    onMouseOver() {
+      sessionStorage.setItem("assigneeID", this.assignee.id);
```

Comment:
I'm still not really a fan of using the sessionStorage, but I guess we'll just let it slide in favor of the feature.